pkg/kubelet/kubelet_volumes.go

```diff
@@ -73,6 +73,27 @@ func (kl *Kubelet) podVolumesExist(podUID types.UID) bool {
 	return false
 }
 
+// podVolumePathsExist checks with the volume manager and returns true any of the
+// volumes for the specified pod are mounted or any of the volume paths of the specified pod exist.
+func (kl *Kubelet) podVolumePathsExist(podUID types.UID) bool {
+	if mountedVolumes :=
+		kl.volumeManager.GetMountedVolumesForPod(
```

Comment:
this is also checked in podVolumesExist() function above?

Reply:
Yes, I didn't want to create a coupling between these two methods `podVolumesExist` and `podVolumePathsExist` even though all current use cases of `podVolumePathsExist` also use the `podVolumesExist`.

Should I remove this check from `podVolumePathsExist`?